### ISSUE TITLE
Window resize ADOBE

### ISSUE THREAD

**Issue by ejakubowska** [rxhanson/Rectangle#1256]
- [ ] This is a new bug that hasn't been documented in a closed issue or in the Readme.

macOS version: macOS Ventura 13.5.2
Rectangle version: 0.70 (76)
Logs if applicable (In Rectangle menu, hold option, "View Logging..."): 

2023-09-12T22:42:05+02:00: AX sizing proposed: (1440.0, 1533.0), result: (1440.0, 1533.0)
2023-09-12T22:42:05+02:00: AX position proposed: (0.0, 25.0), result: (0.0, 25.0)
2023-09-12T22:42:05+02:00: AX sizing proposed: (1440.0, 1533.0), result: (1440.0, 1533.0)
2023-09-12T22:42:05+02:00: leftHalf | display: (0.0, 62.0, 2880.0, 1533.0), calculatedRect: (0.0, 25.0, 1440.0, 1533.0), resultRect: (0.0, 25.0, 1440.0, 1533.0), srcScreen: Studio Display, destScreen: Studio Display, resultScreen: Studio Display
2023-09-12T22:42:09+02:00: Window under cursor fallback matched: Adobe Illustrator 2023 WindowInfo(id: 52, frame: (0.0, 25.0, 1440.0, 1533.0), pid: 1240, processName: "Adobe Illustrator 2023")
2023-09-12T22:42:10+02:00: Window under cursor fallback matched: Adobe Illustrator 2023 WindowInfo(id: 52, frame: (0.0, 25.0, 1440.0, 1533.0), pid: 1240, processName: "Adobe Illustrator 2023")

Hi!
Recently, when I'm using Adobe programs, such as: Photoshop, Illustrator my window is "self" resizing after, for example using zooming. I've read recent bug with Photoshop, but there were no log after click on layer. In my case – there is same log.

Any ideas? Is it Adobe problem or Rectangle?

**Comment by rxhanson** [comment]
Closing this one out as a duplicate of https://github.com/rxhanson/Rectangle/issues/1221

TLDR; this issue is on Adobe's side, unfortunately.
